docs/using-radicle/contributing.md

```diff
@@ -24,41 +24,58 @@ To create your own view of a project, you can use the **Fork** option within the
 local machine and publish your version to the Radicle network as a new project.
 
+<!-- TODO: Update screenshot to have patches -->
+
 ![Fork Project][fo]
 
 Forking a project will move it from the **Following** tab to the **Projects**
-tab. This is because once you fork a project, you create your own view of
-it that is unique to your Device ID. You can [push changes][pc] to this view via
-the `rad` remote and publish them to the network. If another person has
-added you as a remote (See [Tracking & viewing contributions][tv]), they will be
-able to fetch these changes and merge them into their branches (See [Fetching
-and merging contributions][fm]).
-
-You are able to clone your project with the **Checkout** option on your project
-page.
-
-## Getting a contribution merged
-
-Once you [push changes][pc] to a fork of a project and they've been successfully
-published to the network, they can be replicated by other peers. For another
-peer (e.g. the maintainer of a project) to see your changes, they need to [add
-you as a remote][fm] to their projects. This means that until [social coding features
-are introduced][fq], you'll have to manually communicate your patches to the
-maintainer of a project by sending them your Device ID. This can be done through
-email, a developer chat, or social media channel. 
+tab. This is because once you fork a project, you create your own view of it
+that is unique to your Device ID. You can [push changes][pc] to this view via
+the `rad` remote and publish them to the network. You are able to clone your
+project with the **Checkout** option on your project page.
+
+### Creating a patch
+
+Now that you have your own view of the project and you have it checked out, you
+can start creating a patch. In your view of the project you'll see a patches tab next to `Files` and `Commits`. There you see a `New patch` button which will guide you through the steps.
+
+<!-- TODO: Replace with new screenshot -->
+
+![New Patch][np]
+
+First thing you do is make your changes as you always would and commit those
+locally. Once you've done that you have to package them as a patch by creating
+an annotated git tag that starts with `radicle-patch/`, like this:
+
+`git tag--annotate radicle-patch/<name>`
+
+Be sure to replace `<name>` with the name of your patch.
+
+Next git will allow you to add a title and description to the tag, those will be
+the ones that show up in the UI, so be sure to add those to make it easy for the
+maintainer to understand the changes you're proposing.
+
+<!-- TODO: Replace with screenshot showing the tag message thing -->
+
+![Tag Message][tm]
+
+All that's left to do is to publish the new patch by simply pushing the git tag to the network: `git push --tags rad`. Your patch is now public 🎉.
+
+Make sure the maintainer is tracking you so that they see your patch. To learn how to do that, take a look at [tracking and viewing contributions][tv].
```

Comment:
Change:
"…is tracking you so that…"
to
"…has added your Device ID as a remote on their project so that…"